src/panels/config/automation/condition/ha-automation-condition-row.ts

```diff
@@ -238,6 +273,17 @@ export default class HaAutomationConditionRow extends LitElement {
     return [
       haStyle,
       css`
+        .disabled {
+          opacity: 0.5;
+        }
+        .disabled-bar {
+          background: var(--error-color);
```

Comment:
IMHO it's not an error if done intentionally.

```suggestion
          background: var(--warning-color);
```

Reply:
I must agree I've seen better... 

<img width="356" alt="image" src="https://user-images.githubusercontent.com/195327/165775837-bb871960-d56f-43d1-bda4-8e2f48b6fb90.png">

This is how we show disabled in the integrations dashboard, maybe take that?